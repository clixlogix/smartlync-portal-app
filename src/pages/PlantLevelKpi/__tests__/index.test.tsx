import * as React from 'react';
import { render } from '@testing-library/react';
import { HelmetProvider } from 'react-helmet-async';

import { PlantLevelKpi } from '..';

const renderComponent = () =>
    render(
        <HelmetProvider>
            <PlantLevelKpi />
        </HelmetProvider>,
    );

describe('<PlantLevelKpi />', () => {
    it('should match the snapshot', () => {
        const component = renderComponent();
        expect(component.container.firstChild).toMatchSnapshot();
    });
});
