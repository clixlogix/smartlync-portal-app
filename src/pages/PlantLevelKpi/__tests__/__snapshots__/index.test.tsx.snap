// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlantLevelKpi /> should match the snapshot 1`] = `
<div
  class="page-body col"
>
  <div
    class="page-wrapper "
  >
    <iframe
      aria-hidden="true"
      frameborder="0"
      src="about:blank"
      style="display: block; opacity: 0; position: absolute; top: 0px; left: 0px; height: 100%; width: 100%; overflow: hidden; pointer-events: none; z-index: -1;"
      tabindex="-1"
    />
    <iframe
      allowfullscreen=""
      frameborder="0"
      height="-100"
      src="https://app.powerbi.com/reportEmbed?reportId=4c1161f6-8be0-496c-83bf-b025ed69d20f&autoAuth=true&ctid=d4062de4-74ba-4730-a339-59645ae170de&config=eyJjbHVzdGVyVXJsIjoiaHR0cHM6Ly93YWJpLXVzLWVhc3QyLXJlZGlyZWN0LmFuYWx5c2lzLndpbmRvd3MubmV0LyJ9&formatLocale=en-US&language=en-US&filterPaneEnabled=false"
      title="Summary Plant KPIs at a Glance"
      width="0"
    />
  </div>
</div>
`;
